README.md

```diff
@@ -27,20 +27,25 @@ kubectl -n kube-system get deployment coredns -o yaml | \
 
 ### OpenShift
 Follow basic setup [here](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started)
+Use oc from https://apb-oc.s3.amazonaws.com/apb-oc/oc-linux-64bit.tar.gz to be able to correctly deploy openshift 3.11 alpha.
 
 ```bash
-oc cluster up --tag=v3.11
+oc cluster up --tag=v3.11 --public-hostname=$(hostname -f)
 ```
 
 ### OLM
 After deploying a cluster, run OLM:
 
 ```bash
-curl https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/scripts/run_console_local.sh | bash -
+yum install -y jq
+curl https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/scripts/run_console_local.sh -o run_console_local.sh
+chmod +x run_console_local.sh
+./run_console_local.sh &
 ```
 
 ##### Deploy OLM with ansible playbooks
 ```bash
+yum install -y ansible
```

Comment:
Instead of adding `yum install xxx`, let's either provide a requirements.txt or list required packages and version at the top of each section since this list may grow.


##### Requirements
jq
ansible >= 2.4
oc >= 3.11 -> `docker cp $(docker create docker.io/openshift/origin:latest):/usr/bin/oc`